How-To-Load-Data.md

```diff
@@ -0,0 +1,22 @@
+# Table of contents
+- [Populating translation opportunities](#populating-translation-opportunities)
+- [Populating question opportunities](#populating-question-opportunities)
```

Comment:
Add a section on how to populate explorations and classroom also

Reply:
Done